packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/modals/JoinZoomNowModal.tsx

```diff
@@ -0,0 +1,44 @@
+import Modal from 'antd/lib/modal/Modal'
+import JoinZoomButton from '../JoinZoomButton'
+import ReQueuingButton from '../ReQueuingButton'
+import { Popconfirm } from 'antd'
+
+type JoinZoomNowModalProps = {
+  open: boolean
+  zoomLink?: string
+  taName?: string
+  onJoin: () => void
+  setRequeuing: () => void
+}
+
+const JoinZoomNowModal: React.FC<JoinZoomNowModalProps> = ({
+  open,
+  zoomLink,
+  taName,
+  onJoin,
+  setRequeuing,
+}) => {
+  return (
+    <Modal
+      open={open}
+      footer={[]}
+      style={{ top: '30%' }}
+      closable={false}
+      title={`${taName ?? 'A TA'} is Ready for You Now!`}
```

Comment:
```suggestion
      title={`${taName ?? 'A TA'} is ready for you now!`}
```
imo I think sentence case reads a little better, maybe because it puts emphasis on the name, idk. Small thing
![image](https://github.com/user-attachments/assets/ea0769f1-04f6-4a87-9848-006590473a0b)
![image](https://github.com/user-attachments/assets/e03ab603-b4f3-4ef0-b47c-c7e5357608f0)

Reply:
I also experimented with putting a green "!" icon here.
![image](https://github.com/user-attachments/assets/d1d98fee-7f81-4288-b171-117591771215)

I think I like it, but maybe you want to play around with it? Code is here:
```suggestion
      title={<div className='flex items-center gap-x-2'>
        <InfoCircleFilled className='text-green-500 text-2xl' /> {`${taName ?? 'A TA'} is ready for you now!`}
      </div>}
```